tools/nl/svindex_differ/differ.py

```diff
@@ -104,9 +113,28 @@ def run_diff(base_file, test_file, query_file, output_file):
 
 def main(_):
   assert FLAGS.base and FLAGS.test and FLAGS.queryset
-  base_file = _maybe_copy(FLAGS.base)
-  test_file = _maybe_copy(FLAGS.test)
-  run_diff(base_file, test_file, FLAGS.queryset, _REPORT)
+  base_file = _maybe_copy_embeddings(FLAGS.base)
+  test_file = _maybe_copy_embeddings(FLAGS.test)
+
+  test_model_path = ""
+  if "ft" in test_file:
+    # This means we are using embeddings built on finetuned model.
+    # Download the model if needed.
+
+    # Extract the model name.
+    # test embeddings name is of the form:
+    #  <embeddings_size_*>.<ft_final_*>.<ft_intermediate_*>.<base_model>.csv
```

Comment:
Is that true when you generate final directly from base, as is the current plan?

Reply:
Yup, still true. Updated the function comment to reflect that.